activejob/lib/active_job/serializers/object_serializer.rb

```diff
@@ -43,7 +41,7 @@ def deserialize(_argument)
         raise NotImplementedError
       end
 
-      protected
+      private
```

Comment:
I want this to be protected because I want the `klass` method to be documented.

Reply:
What do you think about expressing this method as private and add `# :doc:` for `klass`. Looks like we prefer this approach.

Reply:
Good idea!